sdk/resourcemanager/azure-resourcemanager-appplatform/src/main/java/com/azure/resourcemanager/appplatform/implementation/SpringAppImpl.java

```diff
@@ -145,7 +145,6 @@ private void ensureProperty() {
         }
     }
 
-    @Override
     public SpringAppImpl withDefaultPublicEndpoint() {
```

Comment:
Reason of this change?

Reply:
Missed it in conflict resolution, added back.